packages/webdriverio/src/commands/element/moveTo.ts

```diff
@@ -26,10 +28,20 @@ export async function moveTo (
     if (!this.isW3C) {
         return this.moveToElement(this.elementId, xOffset, yOffset)
     }
+    const isIntoView = async () => {
+        return await browser.execute(isElementInViewportScript, {
+            [ELEMENT_KEY]: this.elementId, // w3c compatible
+            ELEMENT: this.elementId // jsonwp compatible
+        } as any as HTMLElement)
+    }
     /**
      * W3C way of handle the mouse move actions
      */
     const browser = getBrowserObject(this)
+    await  this.scrollIntoView({ block : 'nearest', inline: 'nearest', behavior: 'instant' })
```

Comment:
```suggestion
    await this.scrollIntoView({ block : 'nearest', inline: 'nearest', behavior: 'instant' })
```

Reply:
deleted in https://github.com/webdriverio/webdriverio/pull/11565/commits/44f648b9b9a4a145063331541ccf96285badcd2f